modules/nf-core/multiqc/main.nf

```diff
@@ -1,14 +1,16 @@
 process MULTIQC {
```

Comment:
Doesn't these changes to `modules/nf-core/multiqc/main.nf` mean that the sha of `multiqc` in `modules.json` should be updated?

Reply:
sha has been updated actually